test/runtests.jl

```diff
@@ -46,6 +46,8 @@ isordered(::Type{T}) where {T<:AbstractDict} = false
     callsites = find_callsites_by_ftt(iterate, (Base.IdSet{Any}, Union{}); optimize=false)
     @test callsites[1].info isa Cthulhu.ConstPropCallInfo
 
+    @test Cthulhu.get_effects(callsites[1].info) isa Cthulhu.Effects
```

Comment:
Maybe we can do this check within `find_callsites_by_ftt` instead, which would then function as a basic test for `CallInfo` interface.

At each test case, we may want to focus on the correctness of collected effects information, rather than checking the basic interface requirement.

Reply:
Done (as far as we can for now)